CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+-   for express framework, while fetching the existing response header, use `get` function instead of `getHeaders`
```

Comment:
Add a title for this like:
```
### Changed
- for express framework, while fetching the existing response header, use `get` function instead of `getHeaders`
```